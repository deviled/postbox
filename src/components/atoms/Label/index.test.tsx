import React, { FC } from 'react';
import { render } from '__tests__/utils';
import { Label } from '.';

describe('Label.tsx', () => {
  const Component: FC = () => (
    <Label>
      Name
      <input type="text" name="name" />
    </Label>
  );

  it('should match snapshot', () => {
    const { baseElement } = render(<Component />);

    expect(baseElement).toMatchSnapshot();
  });
});
