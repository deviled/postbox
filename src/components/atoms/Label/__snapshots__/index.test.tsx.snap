// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label.tsx should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-bottom: 16px;
}

.c0:last-of-type {
  margin-bottom: 24px;
}

<body>
  <div>
    <label
      class="c0"
    >
      Name
      <input
        name="name"
        type="text"
      />
    </label>
  </div>
</body>
`;
